using-timescaledb/continuous-aggregates.md

```diff
@@ -156,18 +156,22 @@ ALTER VIEW device_summary SET (timescaledb.refresh_interval = '10 min');
 ```
 Which sets the interval at which the view refreshes to 10 minutes.
 
-
 Other alterations to the continuous aggregate view are currently disallowed. To
 alter a continuous aggregate view in other ways it must be dropped and
 re-created; this can entail some time to re-calculate aggregations.
 
-A continuous aggregate may be dropped by using the `DROP VIEW` command, the
-`CASCADE` parameter is required.
+A continuous aggregate may be dropped by using the `DROP VIEW` command, which
+deletes the hypertable that comprises the (materialized) continuous aggregate
+view.  The `CASCADE` parameter is required for this command.
+
 ```sql
 DROP VIEW device_summary CASCADE;
 ```
->:TIP: `CASCADE` will only drop those objects that depend on the continuous aggregate,
-it does not drop the underlying hypertable or delete any data.
+
+>:TIP: `CASCADE` will only drop those objects that depend on the continuous
+aggregate, it does not drop or otherwise delete from the underlying hypertable
```

Comment:
I would say, including the data materialized in the continuous aggregation, but does not drop or delete data from the underlying hypertable.